backend/LexCore/Auth/LexAuthConstants.cs

```diff
@@ -8,6 +8,7 @@ public static class LexAuthConstants
     public const string IdClaimType = "sub";
     public const string ProjectsClaimType = "proj";
     public const string EmailUnverifiedClaimType = "unver";
+    public const string CanCreateProjectClaimType = "mkProject";
```

Comment:
Not super important, but this name is pretty inconsistent if you compare it to `unver` (no camel case) and `proj` (project abbreviated). You could go with camel-case for everything too.
```suggestion
    public const string CanCreateProjectClaimType = "mkproj";
```